_posts/2020-07-02-july-meetup.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "July 2020"
+date:   2020-07-02 00:00:00
```

Comment:
You'll have to make the date be in the past (same for the filename) so that GitHub Pages picks it up. Otherwise it thinks it's supposed to be published some time in the future. 

So just use today's date instead, it won't show up anywhere anyway.